public/app/controllers/resources_controller.rb

```diff
@@ -243,6 +243,7 @@ def tree_node_from_root
 
   def inventory
     uri = "/repositories/#{params[:rid]}/resources/#{params[:id]}"
+    check_staff_access(uri)
```

Comment:
@avatar382 if I delete this line (and remove the `check_staff_access` method altogether) then the functionality is exactly the same because the visibility is determined by the `smart_staff_links` js. So I think this can be chopped out, right?

Reply:
@mark-cooper Yes -- good catch, I added that while trying to figure out how the the logged in access works.

I've removed all references to the `check_staff_access` method and rebased with the latest master.